app/routes/endemics/index.js

```diff
@@ -54,13 +56,13 @@ module.exports = {
         session.setEndemicsClaim(request, referenceKey, tempClaimId)
 
         // new user
-        if ((!Array.isArray(claims) || !claims?.length) && latestVetVisitApplication === undefined) {
+        if ((claims.length === 0) && latestVetVisitApplication === undefined) {
```

Comment:
Is claims ever undefined? Just because I think it will fail on the .length if it ever can be

Reply:
No we always return an array, catching 404s in the error handler at the moment but need to fix the endpoint in ffc-ahwr-application really. 

https://github.com/DEFRA/ffc-ahwr-farmer-claim/pull/404/files#diff-d41e3c3a63b911a844378e8760c201f4dd18ff5df27e2f58bf95fc700bf279eaR16